crates/transport-http/Cargo.toml

```diff
@@ -43,8 +43,13 @@ hyper-tls = { workspace = true, optional = true }
 alloy-rpc-types-engine = { workspace = true, optional = true }
 jsonwebtoken = { workspace = true, optional = true }
 
+# trace propagation
+opentelemetry-http = { version = "0.31.0", optional = true}
+tracing-opentelemetry = { version = "0.32.0", optional = true}
+opentelemetry = { version = "0.31.0", optional = true }
+
 [features]
-default = ["reqwest", "reqwest-default-tls"]
+default = ["reqwest", "reqwest-default-tls", "traceparent"]
```

Comment:
nit: `trace-parent` is less easy to misread as `transparent` which has specific meaning in Rust core & Serde